### ISSUE TITLE
Data preparation

### ISSUE THREAD

**Issue by dveni** [dveni/ssl-wearable-data-clf#6]
- [ ] Interpolation
- [ ] Noramlization

Data preparation done by the authors:

Minimal pre-processing was applied to the raw sensor datasets. The data was first normalized by applying
z-normalization using the mean and standard deviation of the training data for each of the sensor channels. Each
dataset is then segmented into sliding windows with size 400 × 3, where 400 is the number of time stamps and 3
being the number of channels of a triaxial accelerometer. The sliding windows share 50% overlap, where the next
window has a starting-time shifted 200 timestamps to the future. Data from 20% − 25% of users from each labeled
dataset was kept unseen to the models and used as the test set in order to evaluate the generalizability of the
models. There is no re-sampling performed to standardize the sampling frequencies of the datasets. The same
configurations and pre-processing procedures are kept across different datasets because we want to highlight that
the performance achieved is a result of the training paradigm but not because of the differences in configurations
or pre-processing procedures. Furthermore, a similar pre-processing protocol was also adopted in previous work
[39] and, to make for a fair comparison, we followed similar pre-processing steps.
